### PR TITLE
2022.01.1

### PR DESCRIPTION
**Pull request by kumoy**
Fixes # .

Changes offered/proposed in this pull request:
- 
- 
- 

* New PR Alert to: @plus3it/spel

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+**Released**: 2022.01.18
+
+**Commit Delta**: [Change from 2021.12.1 release](https://github.com/plus3it/spel/compare/2021.12.1...2022.01.1)
+
+**Manifests**: <https://github.com/plus3it/spel/blob/2022.01.1/manifests>
+
+**Summary**:
+
+*   Publishes image for CentOS 8 Stream
+*   Removes image CentOS 8
```
Comment (eemperor) [inline CHANGELOG.md:14]: I think we're still keeping the previously published old CenOS 8 AMIs.  Just not publishing new CentOS 8.

Comment (lorengordon) [reply CHANGELOG.md:14]: Yeah, let's say something like:
```suggestion
*   Deprecates CentOS 8 image -- published `minimal-centos-8-hvm` images will remain available for the time being
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+*   "Extra" packages updated in this release:
+    - amazon-ssm-agent-3.1.804.0-1
+    - aws-cli/1.22.37
+    - aws-cli/2.4.11
```
Comment (lorengordon) [inline CHANGELOG.md:18]: This should include:

- aws-cfn-bootstrap (2.0)
- ec2-hibinit-agent-1.0.2-3
- ec2-instance-connect-1.1-15
- ec2-net-utils-1.5-3

And we'll need a bullet on the aws/amazon/ec2 packages that are being removed by default, and provided instead in the `spel` yum repo, enabled by default. So anyone that needs them can use `yum install ...`

**Comment by lorengordon** [comment]
@kumoy One more bullet we should add is a note about how we are now maintaining a yum repo where we are publishing the amazon/aws/ec2 packages. A bare minimum of the packages are pre-installed, aligning with the pre-installed packages from Amazon Linux 2. Other packages are available via the `spel` repo, and can be installed using `yum install ...`. The `spel` repo itself can be installed using the `spel-release` package. The repo can be browsed at https://spel-packages.cloudarmor.io.

**Review by lorengordon — APPROVED** [review]
rock and roll 👍

**Pull request merged**